presto-iceberg/src/main/java/com/facebook/presto/iceberg/HiveTableOperations.java

```diff
@@ -206,6 +207,10 @@ public TableMetadata refresh()
             throw new UnknownTableTypeException(getSchemaTableName());
         }
 
+        if ("true".equalsIgnoreCase(table.getParameters().get(PRESTO_VIEW_FLAG))) {
```

Comment:
Are these test failures from previous commits? If so, these fixes need to be in the same commit that introduces the test failure. A general good practice is that there should not be any commit breaks the tests.